subgraphs/compound-v2/README.md

```diff
@@ -0,0 +1,23 @@
+# Compound Subgraph
+
+
+## Notes
+
++ Each CToken is a different "Market" > need to import all of those
++ Figure out how to spin up locally
++ OpenZepplin subgraph resource: https://docs.openzeppelin.com/subgraphs/0.1.x/
+
+
+## Links
+
+Protocol: https://compound.finance/
+
+Docs: https://compound.finance/docs
+
+Smart contracts: https://github.com/compound-finance/compound-protocol
+
+Deployed addresses: https://compound.finance/docs#networks
+
+Existing subgraphs: https://github.com/compound-finance/compound-v2-subgraph
```

Comment:
This is a more up-to-date subgraph: https://github.com/graphprotocol/compound-V2-subgraph

Reply:
thanks- updated